NINJA-config.sh

```diff
@@ -94,8 +94,7 @@ main() {
   echo DEPS prebuilt/ninja/*/deps.txt
 
   echo
-  # TODO: remove --without-readline once frontend_pyreadline is complete.
-  _OIL_DEV=1 ./configure --without-readline
+  ./configure
```

Comment:
OK it looks like we need `_build/detected-config.sh`

I might add back the guard to remove the `.h` file, because we have a separate CPP `.h` file that is tiny

The other `.h` file annoys me because it takes awhile to generate, and is part of the CPython stuff